core/templates/components/ck-editor-helpers/ck-editor-copy-content-service.ts

```diff
@@ -34,14 +34,18 @@ interface CkEditorCopyEvent {
 export class CkEditorCopyContentService {
   private readonly OUTPUT_VIEW_TAG_NAME = 'ANGULAR-HTML-BIND';
   private readonly NON_INTERACTIVE_TAG = '-noninteractive-';
+  // Allows skillreview components to be copied as well by making them
+  // a part of the allowed widgets.
```

Comment:
This comment is not required as the list is very generic and it's self-explanatory! :)

Reply:
Will remove.